### ISSUE TITLE
Migrate Tailscale to official k8s setup

### ISSUE THREAD

**Issue by ankushg** [mvisonneau/helm-charts#26]
Tailscale seems to have some semi-official docs for getting set up with k8s, including a Dockerfile and support for reading auth from a Secret. 

They don't publish the k8s container image or have a helm chart though! 

It might be worth reducing your maintenance overhead by aligning your helm chart with their code

https://github.com/tailscale/tailscale/tree/main/docs/k8s

**Comment by mvisonneau** [comment]
absolutely, I am more than happy to work on it as this would be preferable for the community. I believe this is likely something in the pipe on their end though. @josharian would you happen to know what the plans are?

**Comment by josharian** [comment]
@maisem over to you
